src/core/namespace.c

```diff
@@ -1689,6 +1739,46 @@ int bind_mount_add(BindMount **b, size_t *n, const BindMount *item) {
         return 0;
 }
 
+void mount_images_free_many(MountEntry *p, size_t n) {
+        assert(p || n == 0);
+
+        for (size_t i = 0; i < n; i++)
+                mount_entry_done(&p[i]);
+
+        free(p);
```

Comment:
This should return (MountEntry*) NULL. Just use `mfree` here.

Reply:
ok, following the same pattern I'll also zero the counter in the function